__DEFINES/lighting.dm

```diff
@@ -66,3 +66,9 @@
 #define LIGHT_COLOR_SLIME_LAMP "#AFC84B" //Weird color, between yellow and green, very slimy. rgb(175, 200, 75)
 #define LIGHT_COLOR_TUNGSTEN   "#FAE1AF" //Extremely diluted yellow, close to skin color (for some reason). rgb(250, 225, 175)
 #define LIGHT_COLOR_HALOGEN    "#F0FAFA" //Barely visible cyan-ish hue, as the doctor prescribed. rgb(240, 250, 250)
+
+//Defines for lighting status, see power/lighting.dm
+#define LIGHT_OK     0
```

Comment:
I mean, this file is more for the lighting system, not lighting fixtures.

But it's fine I guess.